no-computer.md

```diff
@@ -5,51 +5,64 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+The project is for users that live in dorm suites  and cannot decide what movies to watch together. A language would a be good way to solve their needs because it would structurally organize arguments in what to pick to watch. This would be much better than having suitemates bickering over a movie.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need to readily solve the problem of picking movies is met by the idea. We are helping people in college who live in dorm suites that need to decide what movies to watch. Current experience can involve loud arguing and bickering. The experience can be improved to reduce arguing and making everyone friendlier.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would be appropriate because it would be a way for others to communicate with each other rationally. The need is addressed by using a language if everyone believes the language is fair (which will hopefully be true).
```

Comment:
Pretty cool idea, though it seems like some kind of advanced movie review website where dorm mates can agree and enter parameters (like genre, length, year), and then vote on a subset of movies might be more effective though.